scripts/pyperf_env.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+set -euo pipefail
+
+if [ "$#" -gt 1 ]; then
+    echo "Too many arguments"
+    exit 1
+elif [ "$#" -eq 0 ]; then
+    with_staticx=""
+elif [ "$1" == "--with-staticx" ]; then
+    with_staticx="$1"
+else
+    echo "Unexpected argument: $1"
+    exit 1
+fi
+
+if [ "$(uname -m)" = "aarch64" ]; then
+    exit 0;
+fi
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    curl \
+    build-essential \
+    iperf llvm-12-dev \
+    clang-12 libclang-12-dev \
+    cmake \
+    python3 python3-pip \
+    flex \
+    libfl-dev \
+    bison \
+    libelf-dev \
+    libz-dev \
+    liblzma-dev \
+    ca-certificates \
+    git \
+    patchelf scons
+
+if [ -n "$with_staticx" ]; then
+    apt-get install -y --no-install-recommends \
+        python3-pip ca-certificates git \
+        patchelf scons
```

Comment:
These are all already installed by the previous installer, so remove from either location?

Reply:
👍